pallets/exchange-rate/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "exchange-rate"
+version = "0.1.0"
+authors = ["Ilya Beregovskiy"]
+edition = "2018"
+
+[features]
+default = ['std']
+std = [
+    'codec/std',
+    'frame-support/std',
+    'frame-system/std',
+    'safe-mix/std',
+    'pallet-balances/std',
+    'pallet-randomness-collective-flip/std',
+    'pallet-transaction-payment/std',
+    'sp-core/std',
+    'sp-io/std',
+    'sp-runtime/std',
+    'sp-std/std',
+
+    'roaming-operators/std',
+]
+
+[dependencies]
+codec = { version = '1.2.0', package = 'parity-scale-codec', default-features = false, features = ['derive'] }
+safe-mix = { version = '1.0.0', default-features = false }
+frame-support = { version = '2.0.0', default_features = false }
+frame-system = { version = '2.0.0', default_features = false }
+pallet-balances = { version = '2.0.0', default_features = false }
+pallet-randomness-collective-flip = { version = '2.0.0', default_features = false }
+pallet-transaction-payment = { version = '2.0.0', default_features = false }
+sp-core = { version = '2.0.0', default_features = false }
+sp-io = { version = '2.0.0', default_features = false }
+sp-runtime = { version = '2.0.0', default_features = false }
+sp-std = { version = '2.0.0', default_features = false }
+
+roaming-operators = { default_features = false, package = 'roaming-operators', path = '../roaming/roaming-operators' }
```

Comment:
FYI, the only reason i've been importing this into all the pallets is because it was the only way i figured out how to get access to the `Balance` type, so if you can think of a better way that'd be great, otherwise we can solve later on all at once